src/main/java/it/gov/pagopa/pu/processecexutions/model/exportfile/PaidExportFileFilter.java

```diff
@@ -12,6 +12,6 @@
 @AllArgsConstructor
 public class PaidExportFileFilter implements ExportFileFilter {
 
-  private LocalDateIntervalFilter paymentDate;
+  private OffsetDateTimeIntervalFilter paymentDate;
```

Comment:
@Giuseppe-LaManna I remember that when you implemented this, the requirement was to filter using a LocalDate ignoring the dateTime.
@pelliccm is this requirement changed?

Eventually, we should also rename the field in order to match the model queried field `paymentDateTime` instead of `paymentDate`.